packages/opentelemetry-cloud-trace-propagator/test/CloudPropagator.test.ts

```diff
@@ -179,6 +191,9 @@ describe('CloudPropagator', () => {
           'd4cda95b652f4a1592b449d5929fda1b/foo7929822056569588882bar;o=1',
         too_long_id:
           '111111111111111111111111111111111111111111111/7929822056569588882;o=1',
+        too_short_id: '1111111111111/7929822056569588882;o=1',
+        invalid_flag:
+          'd4cda95b652f4a1592b449d5929fda1b/7929822056569588882;o=123',
```

Comment:
This flag value is valid, but the resulting context should mask out all bits except bit 0.